### DISCUSSION TITLE
How to expose dev to specific network?

### DISCUSSION THREAD

**Discussion by jasmerah1966** [ditatompel/insights#55] [Q&A] [answered]
Waktu jalanin `npm run dev` listennya ke localhost. Apakah ada fitur atau cara supaya saat melakukan perubahan atau edit bisa diakses dari lokal network?

Karena kadang gw testing pakai remote computer dan akses dari pc yang lain.

Edit supaya lebih jelas:

Contoh kasusnya, network lokal gw `192.168.0.0/24`. PC gw yg gw pake untuk akses browser ada di `192.168.0.2`, sedangkan gue edit project ini pake vscode server yang ada di `192.168.0.3`. Jd gw pinginnya `npm run dev`nya listen ke `192.168.0.3:1313` misalnya. Jd gue bisa akses preview dari pc `192.168.0.2`.

Makasih.

**Comment by ditatompel** [comment, marked as answer]
Bisa passing args (hugo command dalam hal ini) dari npm script.

Misal dari kasus diatas:
```
npm run dev -- --baseURL http://192.168.1.103/ --bind 192.168.1.103
```

Untuk flag lebih lengkap bisa lihat di `hugo server --help`

**Reply by jasmerah1966** [reply]
Sip bisa, thanks!
